cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -132,47 +153,58 @@
       "type": "array"
     },
     "date_created": {
+      "description": "The data-taking year",
```

Comment:
Note that some fields such as `date_created` etc are applicable also to non-dataset records, so that seeing "data-taking year" for a software record might look bizarre.  It may be good to explain this.